src/utils/SetupUtil.js

```diff
@@ -122,6 +114,8 @@ export default {
           throw new Error('Could not determine IP from docker-machine.');
         }
 
+        if (util.isLinux()) { docker.ensureLinuxDockerIsRunning() }
```

Comment:
Sorry for the delay. This sounds good to me - one function for native and one via machine.